documentation/book/getting-started.adoc

```diff
@@ -9,12 +9,10 @@ The Docker images are available on {DockerRepository}.
 
 === Prerequisites
 
-An {ProductPlatformName} cluster is required to deploy {ProductLongName}. {ProductName} supports all kinds of clusters - from public and
-private clouds down to local deployments intended for development purposes. This guide expects that an {ProductPlatformName}
-cluster is available and the
-ifdef::Kubernetes[`kubectl` or]
-`oc` command line tools are installed and configured to connect
-to the running cluster.
+{ProductLongName} runs on {ProductPlatformName} cluster.
+{ProductName} supports ifdef::Kubernetes[{KubernetesLongName} {KubernetesVersion}] or {OpenShiftLongName} {OpenShiftVersion}.
+{ProductName} works on all kinds of clusters - from public and private clouds down to local deployments intended for development purposes.
```

Comment:
I love this, very clever :). Its grammatically sound and I think it would flow even better if we dropped the word "purposes".

Reply:
The _grammatically sound_ parts are from Tom :-)